README.md

```diff
@@ -202,71 +220,64 @@ take the unencrypted value from the first_name model field as the input value
 to create the hash. If you did not specify an original attribute, the field 
 would work as it does now and would remain backwards compatible.
 
-#### Decryption using custom model managers
+##### PGP fields
+
+When accessing the field name attribute on a model instance we are getting the
+decrypted value.
 
-If you use the bundled `PGPManager` with your custom model manager, all encrypted 
-fields will automatically decrypted for you (except for hash fields which are one 
-way).
+Example:
+```
+>>> # When using a PGP public key based encryption
+>>> my_model = MyModel.objects.get()
+>>> my_model.value
+'Value decrypted'
+```
 
-N.B. The bundled manager does not support decryption of fields from FK joins. For 
-example if the `MyModel` class had a FK to to `AnotherModel` class, no encrypted 
-fields be decrypted in the joined `AnotherModel`.
+Filtering encrypted values is now handled automatically as of 2.4.0. And `aggregate`
+methods are not longer supported and have been removed from the library.
 
-It is recommended that you use the bundled `PGPAdmin` class if using the custom 
-model manager and the Django Admin. The Django Admin performance suffers when 
-using the bundled custom manager. The `PGPAdmin` disables automatic decryption 
-for all ORM calls for that admin class.
+Also, auto-decryption is support for `select_related()` models. In your manager, be
+sure to set `use_for_related_fields = True` to enable auto-decryption.
```

Comment:
Is `use_for_related_fields` necessary? It looks like [Manager.use_for_related_fields was deprecated in 1.10](https://docs.djangoproject.com/en/2.1/releases/1.10/#manager-use-for-related-fields-and-inheritance-changes)

Reply:
Agreed. Didn't realize that had been deprecated. Fixed in next commit.